contracts/DCAHubSwapper/DCAHubSwapperSwapHandler.sol

```diff
@@ -87,6 +86,47 @@ abstract contract DCAHubSwapperSwapHandler is DeadlineValidation, DCAHubSwapperP
     _swapExecutor = _NO_EXECUTOR;
   }
 
+  /// @inheritdoc IDCAHubSwapperSwapHandler
+  function swapWithDexes(SwapWithDexesParams calldata _parameters) external payable returns (IDCAHub.SwapInfo memory) {
+    return _swapWithDexes(_parameters, false);
+  }
+
+  /// @inheritdoc IDCAHubSwapperSwapHandler
+  function swapWithDexesByMeanKeepers(SwapWithDexesParams calldata _parameters) external payable returns (IDCAHub.SwapInfo memory) {
```

Comment:
I'm not a fan of the name either, but I'm not sure how to call it haha

@0xged , any ideas?